client/src/components/notes/notes.js

```diff
@@ -35,6 +37,14 @@ class Notes extends Component {
     return (
       <div>
         <p>NOTE PLACEHOLDER</p>
+        <Query query={NOTES_AND_ASSOCIATED_TAGS}>
+          {({ loading, error, data }) => {
+            if (loading) return <div>Fetching</div>;
+            if (error) return <div>Error</div>;
+            console.log(data.allNotes.edges.node);
+            return <div>What</div>;
+          }}
+        </Query>
       </div>
     );
```

Comment:
I will take it as given that you will replace the Note Placeholder at some point?